scripts/install_dependencies/install_openvino_dependencies.sh

```diff
@@ -148,14 +148,42 @@ elif [ "$os" == "rhel8" ] ; then
 
     pkgs_core=(libtbb2 libpugixml1v5)
     pkgs_opencv_req=(gtk3)
-    pkgs_python=(python3 python3-pip)
```

Comment:
why python is removed here? it's here in 2022.2 branch

Reply:
@ilya-lavrenov it is added back now. It was a mistake.

Reply:
see the comment below